Source/Core/TerrainCompression.js

```diff
@@ -0,0 +1,35 @@
+/*global define*/
+define([
+        './freezeObject'
+    ], function(
+        freezeObject) {
+    "use strict";
+
+    /**
+     * This enumerated type is used to determine how the vertices of the terrain mesh are compressed.
+     *
+     * @namespace
+     * @alias TerrainCompression
+     *
+     * @private
+     */
+    var TerrainCompression = {
```

Comment:
Perhaps we should use that terminology throughout so things aren't confusing, for example, when Cesium supports both mesh compression and quantization glTF extensions.